src/XRP/issued-currency-client.ts

```diff
@@ -94,6 +106,67 @@ export default class IssuedCurrencyClient {
     return trustLines
   }
 
+  /**
+   * Returns information about the total balances issued by a given account,
+   * optionally excluding amounts held by operational addresses.
+   * @see https://xrpl.org/issuing-and-operational-addresses.html
+   *
+   * @param account The account for which to retrieve balance information, encoded as an X-Address.
+   * @param hotwallet (Optional) An operational address to exclude from the balances issued, or an array of such addresses,
+   *                   encoded as X-Addresses.
+   * @see https://xrpaddress.info/
+   * @returns A GatewayBalances object containing information about an account's balances.
+   */
+  public async getGatewayBalances(
+    account: string,
+    hotwallet?: string | Array<string>,
```

Comment:
quick q - why an array? is this a common pattern that we expect? 

`hotwallet` is a SUPER opinionated term, which isn't bad, but...will need to think more about this. could this be `exclude`?

Reply:
The array is simply how the JSON RPC handles it, because there may be several addresses that you want to exclude from your notion of balances.  I think you're right though that hotwallet is very opinionated and not even that clear.  I went ahead and renamed this to `addressesToExclude` - a bit more cumbersome but certainly more clear.